### ISSUE TITLE
ng-if-else/ng-if-else-then use ng-container

### ISSUE THREAD

**Issue by cgatian** [BeastCode/VSCode-Angular-TypeScript-Snippets#37]
snippet: `ng-if-else`

template: 
```
<div *ngIf="expression; else elseTemplate">
  
</div>
<ng-template #elseTemplate>
  
</ng-template>
```
Instead of using a DIV use `ng-container` Reason, when the expression is truthy an additional DIV will be outputted. When Else or Then this DIV is omitted.

**Comment by BeastCode** [comment]
Good point! I like, fix coming in the next build.

thanks /Mike

**Comment by BeastCode** [comment]
Fixed in 4.3.10
